web/app/scripts/social-costs/social-costs-directive.js

```diff
@@ -5,11 +5,35 @@
     function SocialCosts() {
         var module = {
             restrict: 'EA',
-            templateUrl: 'scripts/social-costs/social-costs.html',
+            templateUrl: function(elem, attrs) {
+                // Note that use of templateUrls is discouraged in most cases
```

Comment:
Yeah, that's a good point -- my initial reasoning was that they have very similar behavior, so doing it this way would avoid the boilerplate that would accompany defining a second directive. Now that I've got it all set up, there are a couple places where the functionality is slightly different, so I think this is probably a good change to make.